packages/orbit-components/src/HorizontalScroll/README.md

```diff
@@ -33,6 +33,8 @@ After adding import into your project you can use it simply like:
 | onOverflow        | `() => void`                |          |                      | callback function, fires, if content is overflowed                            |
 | arrows            | `boolean`                   |          |                      | show arrows                                                                   |
 | arrowColor        | `string`                    |          |                      | set arrows color                                                              |
+| arrowLeftTitle    | `string`                    |          |                      | set title for left arrow                                                      |
+| arrowRightTitle   | `string`                    |          |                      | set title for right arrow                                                     |
```

Comment:
while editing this file, could you also take a look 👁️  at `#Spacing` and `#ScrollSnap` anchors? they're not working properly on [doc page](https://marco-horizonal-scroll-a11y.orbit.pages.dev/components/layout/horizontalscroll/react/). I guess make the anchor link lowercase will help.